shanoir-ng-front/src/app/datasets/dataset/common/dataset.common.component.ts

```diff
@@ -51,15 +51,15 @@ export class CommonDatasetComponent implements OnChanges {
             private formBuilder: FormBuilder) {
 
         this.exploredEntityOptions = ExploredEntity.options;
-        this.datasetModalityTypes = DatasetModalityType.options;
+        this.datasetModalityTypes = ['Calibration' , 'Ct' , 'Eeg' , 'Meg' , 'Mesh' , 'Mr', 'ParameterQuantification' , 'Pet' , 'Registration' , 'Segmentation' , 'Spect'  , 'Statistical' , 'Template']
```

Comment:
I have a proposition in order to avoid listing those dataset types here.

* rename `datasetModalityTypes` to `datasetTypes` as those are two different enums
* replace the content of `datasets/shared/dataset-type.model.ts` with : 
```
export const DatasetTypes = [
  'Calibration',
  'Ct',
  'Eeg',
  'Meg',
  'Mesh',
  'Mr',
  'ParameterQuantification',                  
  'Pet',
  'Registration',
  'Segmentation',
  'Spect',
  'Statistical',
  'Template'
] as const;

export type DatasetType = typeof DatasetTypes[number];
```
* `datasetModalityTypes: Option<string>[];` could be `datasetTypes: Option<DatasetType>[];`
* `this.datasetTypes = DatasetTypes.map(prop => new Option<DatasetType>(prop, prop as DatasetType));`

Reply:
What is the difference between datasetType & datasetModalityType ?